src/main/java/frc/robot/util/PathPlannerCommands.java

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.util;
+
+import com.pathplanner.lib.commands.FollowPathHolonomic;
+import com.pathplanner.lib.path.PathPlannerPath;
+import com.pathplanner.lib.util.HolonomicPathFollowerConfig;
+import com.pathplanner.lib.util.ReplanningConfig;
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.DriverStation.Alliance;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.subsystem.DriveSubsystem;
+import frc.robot.subsystem.PositioningSubsystem;
+
+/** Add your docs here. */
+public class PathPlannerCommands {
+    private static final ReplanningConfig REPLANNING_CONFIG = new ReplanningConfig();
+
+    public static Command getFollowPathCommand(
+            DriveSubsystem drive,
+            PositioningSubsystem positioning,
+            PathPlannerPath path,
+            boolean shouldAssumeRobotIsAtStart) {
+        if (DriverStation.getAlliance().map(a -> a == Alliance.Red).orElse(false)) {
+            path = path.flipPath();
+        }
+
+        final PathPlannerPath thePath = path;
+
+        FollowPathHolonomic driveControllerCommand = new FollowPathHolonomic(
+                path,
+                positioning::getRobotPose,
+                drive::getLastRobotRelativeSpeeds,
+                drive::driveRobotRelativeSpeeds,
+                new HolonomicPathFollowerConfig(
+                        drive.translationPathController.toImmutable(),
+                        drive.rotationPathController.toImmutable(),
+                        MoPrefs.autoMaxModuleSpeed.get(),
+                        MoPrefs.autoDriveBaseRadius.get(),
+                        REPLANNING_CONFIG),
+                () -> false,
```

Comment:
Again, would be nice to have a comment here.

At first glance, it seems extra to manually flip the path ourselves instead of just letting the `FollowPathHolonomic` command flip it for us.

I recognize it's to support setting the initial robot pose. But a comment would be nice to make it more obvious.